intro2.md

```diff
@@ -0,0 +1,405 @@
+the rest of this document.
+
+### Verifiers
+
+Verifiers monitor L1 for rollup data. They serve three purpose:
+
+1. Serving rollup data to users; and
+2. Verifying rollup integrity and disputing invalid assertions.
+3. Propagate the L2 state among validators (\*)
+
+In order for the network to remain secure there must be at least one honest
+verifier who is able to verify the integrity of the rollup chain & serve
+blockchain data to users.
+
+(\*) It's important to note that a validator does not rely on this data to
+validate the rollup — only access to the L1 chain is required. However, getting
+the most recent L2 state allows validators to serve the state to users, and to
+pre-validate transactions that should soon be posted to L1. L2 state propagation
+uses the same mechanism as L1 state sync, but must additionally include a
+signature from the sequencer to ensure the legitimacy of the data.
+
+## User Transactions and The Sequencer
+
+As a user, once you have [bridged] (\*) some ETH over to Optimistic Ethereum,
+you will probably interact with it though your wallet, such as Metamask. Add the
+network (chainID 10, [JSON-RPC] endpoint `https://mainnet.optimism.io/`) and
+you're good to go.
+
+(\*) L1 -> L2 deposits will be explained later.
+
+[bridged]: https://www.optimism.io/apps/bridges
+[JSON-RPC]: https://github.com/ethereum/execution-apis
+
+When you send a transaction, it is sent to the node called the sequencer. The
+sequencer will run verify your transaction, execute it if valid, and confirm it.
+This happens much faster than on Ethereum mainnet (around ~1s).
+
+There is currently a single sequencer operated by Optimism PBC. We expect future
+versions of this specification to introduce sequencer decentralization. However,
+despite the sequencer being decentralized, trust is not required. As we will
+see, the sequencer posts its results on Ethereum mainnet (henceforth: layer 1 or
+L1), where they can be permissionlessly challenged with a fraud proof. If the
+sequencer temporarily goes down, Optimism will remain live because users are
+able to submit L2 transactions on L1 (of course, this implies the loss of the
+increased throughput and lower fees).
+
+L2 transactions are identical to L1 transactions, except that they use the
+Optimistic L2 state instead of the L1 state. This includes account balances,
+deployed contracts and contract storage.
+
+## L2 Batches and Blocks
+
+Optimistic Ethereum has blocks, however these work slightly differently from L1
+blocks, which are mined by proof-of-work and will soon be decided by
+proof-of-stake.
+
+Optimistic Ethereum has two kind of blocks:
+1. L2 deposit blocks
+2. L2 sequencer blocks
+
+As it confirms transactions, the sequencer accumulates these transaction in a
+"batch". The transactions in such batches will become **_L2 sequencer block_**
+once posted to the Optimistic Ethereum contracts on L1. Please note that there
```

Comment:
Define a block - I think a good definition of block is a contiguous amount of gas. Transaction at the first index has access to more total gas than the transaction at the second index.

Reply:
We should totally define what a block is. Not sure if it should be included here or elsewhere, but we can link to the glossary for sure. I'm not a fan of calling a block a "contiguous amount of gas" because it seems to imply that a single transaction could be split amongst multiple blocks. I'd rather call it a bundle of transactions. Then the gas constraint can be expressed as "the total gas used by each transaction in the block cannot exceed the block's gas limit".